config/initializers/solidus_pay_tomorrow.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+SolidusPayTomorrow.configure do |config|
+  config.base_url = ENV['PAY_TOMORROW_REDIRECT_BASE_URL']
```

Comment:
What do you think about adding the following command at the end of the sandbox script?

`sed -i '' "/Rails.application.configure do/{p;s/.*/  config.hosts << URI.parse(ENV['PAY_TOMORROW_REDIRECT_BASE_URL']).hostname/;}" config/environments/development.rb # rubocop:disable Layout/LineLength`

In this way, all the specified URL will be accepted automatically as incoming connections.

Reply:
Good idea! Will save the ngrok step for the dev.
But then, I think we'd also add -
`export PAY_TOMORROW_REDIRECT_BASE_URL = ngrok-url` sometime before the sed command.

Because if the ENV variable is not set, then the sed command will fail. Now the question is, we'd also have the ngrok-URL before building the sandbox.

Thoughts?

Reply:
Probably we can change it to:

```
if [ -z $PAY_TOMORROW_REDIRECT_BASE_URL]
then
sed -i '' "/Rails.application.configure do/{p;s/.*/  config.hosts << URI.parse(ENV['PAY_TOMORROW_REDIRECT_BASE_URL']).hostname/;}" config/environments/development.rb # rubocop:disable Layout/LineLength
fi
```

In this way, the sed command will be executed only when the variable is set

Reply:
Added in https://github.com/nebulab/solidus_pay_tomorrow/pull/11/commits/6486d40f7c5ddb4f08e98e140af0587406947c29